.buildbot.sh

```diff
@@ -2,6 +2,7 @@
 set -e
 
 CHERIBUILD=~/build
+CHERI_DIR=/home/buildbot/cheri/output
 
 export SSHPORT=10021
```

Comment:
I'm unsure what the status of the port we use for `ssh`, but I guess this will be ironed out in testing.